src/datagrid/DataGridFilter.tsx

```diff
@@ -8,27 +8,29 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as React from "react";
+import React from "react";
 import {classNames} from "../utils";
+import {getXPositionValue} from "./DataGridUtils";
+import {DEFAULT_TRANSFORM_XY_POSITIONS} from "./constants";
 import {ClassNames} from "./ClassNames";
 import {Button} from "../forms/button";
 import {DataGridRow} from "./DataGrid";
 import {DebounceUtils} from "../forms/common/DebounceUtils";
 
 /**
  * Props for DataGridFilter :
- * @param {style} CSS style
- * @param {className} CSS classnames
- * @param {datagridRef} Refrence for DataGrid on which filter will gets apply. We need this to call method which will update datagird rows.
- * @param {columnName} columnName on which filter will apply
- * @param {placeholder} placeholder for string filter input
- * @param {onFilter} Custom filter logic
- * @param {filterType} Type of filter string or custom
- * @param {disabled} boolean value to enable or disable filter
- * @param {debounce} boolean value to apply debounce behaviour
- * @param {debounceTime} number value debounceTime/Delay value in miliseconds
- * @param {position} position of the filter popup
- * @param {defaultValue} defaultValue of the filter
+ * @param {*} style - CSS style
+ * @param {string} className - CSS classnames
+ * @param {*} datagridRef - Reference for DataGrid on which filter will gets apply. We need this to call method which will update datagird rows.
+ * @param {string} columnName - columnName on which filter will apply
+ * @param {string} placeholder - placeholder for string filter input
+ * @param {function} onFilter - Custom filter logic
+ * @param {FilterType} filterType - Type of filter string or custom
+ * @param {boolean} disabled - boolean value to enable or disable filter
+ * @param {boolean} debounce - boolean value to apply debounce behavior
+ * @param {number} debounceTime - number value debounceTime/Delay value in milliseconds
+ * @param {FilterPosition} position - position of the filter popup
+ * @param {*} defaultValue - defaultValue of the filter
```

Comment:
Please revert these comments related changes

Reply:
Sure, I can revert this. but reverting this to previous format like 
`@param {<variable>} <description>` is completely wrong as var name can not take the place of type.
The format suggested by JSDoc is something like `@param {type} <variable>- <description>` 
Please see https://jsdoc.app/tags-param.html and suggest what could be the best course of action.

Reply:
Please follow existing implemented pattern